catalog/spatial/registry/registry-rest-endpoint/src/main/java/org/codice/ddf/registry/rest/endpoint/RegistryRestEndpoint.java

```diff
@@ -150,14 +149,114 @@ public Response viewRegInfoHtml(@PathParam("registryId") final String registryId
                     .build();
         }
 
-        Map<String, Object> registryMap = reportMapBuilder.buildRegistryMap(registryPackage);
+        try {
+            html = registryReportBuilder.getHTMLFromRegistryPackage(registryPackage,
+                    templateLoader,
+                    "report");
+        } catch (IOException e) {
+            String message = "Error in compiling applying report template.";
+            LOGGER.error("Error in compiling applying report template");
```

Comment:
I will log message, I think the 'and ' was accidentally removed. I can add the registry id to the logger message as well